bootstrap

```diff
@@ -130,6 +130,12 @@ while true; do
             export disable_tools="$1"
             shift;
             ;;
+        --max-sample-failures)
+            shift;
+            echo "[WARNING] --max-sample-failures was not used, so seeting to default of $max_sample_failures"
```

Comment:
This seems wrong...

Reply:
yeah I forgot to change this.  I will in the next merge